commcare-support-library/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="org.commcare.commcaresupportlibrary" />
```

Comment:
Think we should be bumping the Android sdk version in support lib `build.gradle` as well to go along with this change.